lib/resources/auditd_conf.rb

```diff
@@ -32,17 +36,7 @@ def read_params
       return @params if defined?(@params)
 
       # read the file
-      file = inspec.file(@conf_path)
-      if !file.file?
-        skip_resource "Can't find file '#{@conf_path}'"
-        return @params = {}
-      end
-
-      content = file.content
-      if content.empty? && !file.empty?
-        skip_resource "Can't read file '#{@conf_path}'"
-        return @params = {}
-      end
+      content = read_file_content(@conf_path)
```

Comment:
May just want to use `@content` here so we don't have to call down again. Another option is to just remove `@content` from the init block and let it be read here.

Reply:
Hi @jquick , thanks for pointing them out.

When they do not call the read_file_content method in the init block, i.e. removing it from their init blocks, they do not skip the test and do report it as failure if a target file is not found since their to_s methods seem to interrupt the exception from methods.

Thus I will remove the extra call and use `@content` of the init block in the method.